app/src/main/java/com/locoquest/app/dto/User.kt

```diff
@@ -2,4 +2,4 @@ package com.locoquest.app.dto
 
 data class User(
     val uid: String,
-    val displayName: String)
+    val displayName: String?)
```

Comment:
Is this set to null and/or not initialized anywhere?

It's best to make variables not nullable types (remove the ?).  Otherwise, you run the risk of a Null Pointer Exception, from a force not null operation (!!).